calebwagner/uteach-ilearn-server
# fixed get meetings by user bug

#### Changes Made
1. Modified file `message.py` to include `scheduler = AppUser.objects.get(user=request.auth.user)` functionality.
   ​
#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-category-sort
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Confirm file modifications are present as indicated above.
